examples/ggez/map.rs

```diff
@@ -0,0 +1,239 @@
+                                let rect = get_tile_rect(tile.tileset, tile.id);
+                                let ts = tile.tileset;
+                                if let Some((batch, ts_size)) = ts_sizes_and_batches.get_mut(&ts.name) {
+                                    let mut dx = x as f32 * self.map.tile_width as f32 + parallax_pan.0 * (layer.parallax_x - 1.0);
+                                    let mut dy = y as f32 * self.map.tile_height as f32 + parallax_pan.1 * (layer.parallax_y - 1.0);
+
+                                    if self.example_animate {
+                                        dx += (ggez::timer::time_since_start(ctx).as_secs_f32() - x as f32 * 0.3 + i as f32 * 0.25).sin() * 20.0;
+                                        dy += (ggez::timer::time_since_start(ctx).as_secs_f32() * 1.25 + y as f32 * 0.3 + i as f32 * 0.25).cos() * 20.0;
+                                    }
+
+                                    batch.add(
+                                        DrawParam::default()
+                                            .src(ggez::graphics::Rect::new(
+                                                rect.0 as f32 / (*ts_size).0 as f32,
+                                                rect.1 as f32 / (*ts_size).1 as f32,
+                                                rect.2 as f32 / (*ts_size).0 as f32,
+                                                rect.3 as f32 / (*ts_size).1 as f32,
+                                            ))
+                                            .dest([
+                                                dx,
+                                                dy,
+                                            ])
+                                            .color(ggez::graphics::Color::from_rgba(
+                                                0xFF,
+                                                0xFF,
+                                                0xFF,
+                                                (layer.opacity * 255.0) as u8,
+                                            )),
+                                    );
+                                }
+                            }
+                        }
+                    }
+
+                    layer_batches.push(ts_sizes_and_batches.into_values().map(|sb| sb.0).collect());
+                }
+                TileLayer::Infinite(_) => {
+                    unimplemented!()
+                }
+            }
+        }
+
+        layer_batches
+    }
+    
+    fn draw_object(object: &tiled::ObjectData, ctx: &mut Context, draw_param: DrawParam) -> GameResult {
+        match &object.shape {
+            tiled::ObjectShape::Rect { width, height } => {
+                let bounds = graphics::Rect::new(object.x, object.y, *width, *height);
+                let shape =
+                    graphics::Mesh::new_rectangle(ctx, graphics::DrawMode::stroke(2.0), bounds, graphics::Color::CYAN)?;
```

Comment:
Note that there is also `Object::get_tile`, which would be nice to support as well. Of course then the object needs to be accessed through `ObjectLayer::get_object` instead of iterating `ObjectLayerData::objects` directly.

Since this might be a bit of a cludge, we could wait with supporting that until #158 is resolved.